docs/ldml/tr35-keyboards.md

```diff
@@ -1076,435 +1294,783 @@ On some architectures, applications may directly interact with keys before they
 > Children: _none_
-> Occurence: required, multiple
+> Occurrence: required, multiple
 >
 > </small>
 
+
 _Attribute:_ `from` (required)
 
-> This attribute follows the `transform/@from` attribute and contains a string of elements. Each element matches one character and may consist of a codepoint or a UnicodeSet (both as defined in UTS#35 section 5.3.3).
+> The `from` attribute consists of an input rule for matching the input context.
+>
+> The `transform` rule and output pattern uses a modified, mostly subsetted, regular expression syntax, with EcmaScript syntax (with the `u` Unicode flag) as its baseline reference (see [MDN-REGEX](https://developer.mozilla.org/docs/Web/JavaScript/Guide/Regular_Expressions)). Differences from regex implementations will be noted.
 
-_Attribute:_ `before`
+#### Regex-like Syntax
 
-> This attribute follows the `transform/@before` attribute and contains the element string that must match the string immediately preceding the start of the string that the @from matches.
+- **Simple matches**
 
-_Attribute:_ `after`
+    `abc` `𐒵`
 
-> This attribute follows the `transform/@after` attribute and contains the element string that must match the string immediately following the end of the string that the `@from` matches.
+- **Unicode codepoint escapes**
 
-_Attribute:_ `order`
+    `\u1234 \u012A`
+    `\u{012a} \u{1234a}`
 
-> This attribute gives the primary order for the elements in the matched string in the `@from` attribute. The value is a simple integer between -128 and +127 inclusive, or a space separated list of such integers. For a single integer, it is applied to all the elements in the matched string. Details of such list type attributes are given after all the attributes are described. If missing, the order value of all the matched characters is 0. We consider the order value for a matched character in the string.
->
-> * If the value is 0 and its tertiary value is 0, then the character is the base of a new run.
-> * If the value is 0 and its tertiary value is non-zero, then it is a normal character in a run, with ordering semantics as described in the `@tertiary` attribute.
-> * If the value is negative, then the character is a primary character and will reorder to be before the base of the run.
-> * If the value is positive, then the character is a primary character and is sorted based on the order value as the primary key following a previous base character.
->
-> A character with a zero tertiary value is a primary character and receives a sort key consisting of:
->
-> * Primary weight is the order value
-> * Secondary weight is the index of the character. This may be any value (character index, codepoint index) such that its value is greater than the character before it and less than the character after it.
-> * Tertiary weight is 0.
-> * Quaternary weight is the same as the secondary weight.
+    The hex escaping is case insensitive. The value may not match a surrogate or illegal character.
+    The form
```

Comment:
This looks unfinished

Reply:
Fixed in upcoming PR, noting that CLDR-16825 proposes removing \u1234 format